src/XrdCrypto/XrdCryptosslAux.cc

```diff
@@ -485,9 +516,9 @@ int XrdCryptosslX509ParseFile(const char *fname,
       rewind(fcer);
       RSA  *rsap = 0;
       if (!PEM_read_RSAPrivateKey(fcer, &rsap, 0, 0)) {
-         DEBUG("no RSA private key found in file "<<fname);
+         DEBUG("no RSA private key found in file");
```

Comment:
Why is it that you are dumbing down the error message? Frankly, I would really want to know what file it is talking about. This error is now rather useless.

Reply:
Well, the `fname` variable no longer exists.  I can make it a requirement for logging-only, if you'd like.

Reply:
Oh, didn't catch the deletion of fname as it was indirect. If the fname is constant and there is another message referring to it then there is no need. If it's always changing (e.g. the name of each crl or ca) then likely it's good to pass it in if only for debugging purposes.

Reply:
Latest commit added back the debugging output as before.  Resolving.